themes/default/content/product/_index.md

```diff
@@ -12,99 +12,116 @@ subheading: |
-                    encapsulate your cloud architectures and best practices.
+                    Use your IDE with interactive documentation, type checking, and statement completion.
+              - title: Write Policy as Code
+                description: |
+                    Write Policy as Code in programming languages to enforce best practices with [Crossguard](/crossguard/).
 
-        - description: Deploy cloud infrastructure and applications together
+        - title: "Create infrastructure automation workflows"
+          sub_title: "Pulumi Automation API"
+          description: |
+            Create workflows that coordinate provisioning, previewing, refreshing, and destroying cloud resources by using the Pulumi engine as a library in your application code.
+          image: "/images/product/automation-api.png"
+          button:
+            text: "Learn more about Automation API"
+            link: "/docs/reference/pulumi-sdk/"
           features:
-              - title: Multiple deployment options
-                icon: rocketship
-                icon_color: salmon
+              - title: 10x productivity
                 description: |
-                    Deploy infrastructure interactively with a CLI, programmatically with Pulumi’s
-                    [Automation API](/automation/), or through your [CI/CD process](/docs/guides/continuous-delivery/).
-
-              - title: Deployments as Code
-                icon: gear
-                icon_color: salmon
+                   Engineers can manage 10x more cloud resources compared to traditional CLI tools.
+              - title: Build custom CLIs
                 description: |
-                    Run deployments from your application code at runtime with [Automation API](/automation/). Create infrastructure APIs, custom platforms, and CLIs.
-
-              - title: Preview and test changes
-                icon: eye
-                icon_color: salmon
+                    Create CLIs that make it easy for your developers to provision prebuilt architectures.
+              - title: Power up your SaaS
                 description: |
-                    Test and validate infrastructure with standard [unit test frameworks](/docs/guides/testing/#unit-testing) and
-                    [integration tests](/docs/guides/testing/integration/). Preview changes before deploying.
+                    Enable your cloud service to dynamically provision and manage cloud resources at scale.
 
-        - description: Manage cloud applications with visibility and controls
+        - title: "Build infrastructure faster with reusable components"
+          sub_title: "Pulumi Packages"
+          description: |
+            Build and reuse higher-level abstractions for cloud architectures with multi-language Pulumi Packages. Distribute the packages through repositories or package managers so your team members can reuse them.
+          image: "/images/product/pulumi-packages.png"
```

Comment:
instead of an image, can you put the code example box? it's from here: https://www.pulumi.com/registry/packages/eks/

Reply:
maybe... let me try and see. the way I have it abstracted, it assumes an image for this section. But will see if I can get the code box to work,.

Reply:
i added in the code box instead.